frontend/src/store.js

```diff
@@ -31,6 +31,13 @@ export default new Vuex.Store({
   },
   getters: {
     students: (state) => state.students,
-    ...nativeLanguageGetters
-  },
+    ...nativeLanguageGetters,
+    screeningStudents: (state) => state.students.filter((student) => {
+      if (student.status.Description == "SCREENING") {
```

Comment:
@lauratsui can you try this?

```javascript
 state.students.filter((student) => student.status.Description === "SCREENING")
```

Reply:
it's shortcut for `if else`

Reply:
Thanks, made the changes in the latest 5th commit